KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift

```diff
@@ -9,34 +9,125 @@
 import UIKit
 import KarhooSDK
 
+enum VehicleTag: String {
+    case electric, hybrid, wheelchair, childSeat, taxi, executive
+    
+    enum CodingKeys: String, CodingKey {
+        case childSeat = "child-seat"
+    }
+    
+    var title: String {
+        switch self {
+        case .electric:
+            return "Electric"
+        case .hybrid:
+            return "Hybrid"
+        case .wheelchair:
+            return "Wheelchair"
+        case .childSeat:
+            return "Child seat"
+        case .taxi:
+            return "Taxi"
+        case .executive:
+            return "Executive"
+        }
+    }
+    
+    var image: UIImage {
+        switch self {
+        case .electric:
+            return UIImage.uisdkImage("electric")
+        case .hybrid:
+            return UIImage.uisdkImage("circle")
+        case .wheelchair:
+            return UIImage.uisdkImage("wheelchair")
+        case .childSeat:
+            return UIImage.uisdkImage("childseat")
+        case .taxi:
+            return UIImage.uisdkImage("circle")
+        case .executive:
+            return UIImage.uisdkImage("star")
+        }
+    }
+}
+
+enum FleetCapabilities: String {
+    case gpsTracking, flightTracking, trainTracking
+         //driverDetails, vehicleDetails
+    
+    enum CodingKeys: String, CodingKey {
+        case gpsTracking = "gps_tracking"
+        case flightTracking = "flight_tracking"
+        case trainTracking = "train_tracking"
+//        case driverDetails = "driver_details"
+//        case vehicleDetails = "vehicle_details"
+    }
+    
+    var title: String {
```

Comment:
Same question as for the VehicleType enum regarding localization

Reply:
Here is the resources for Android:
https://github.com/karhoo/karhoo-android-ui-sdk/blob/18e335e031ee036f62bd36f80efdbdd88217a9ab/uisdk/src/main/res/values/strings.xml#L22

Reply:
added localised strings here now